bundles/org.openhab.binding.tacmi/README.md

```diff
@@ -142,8 +115,50 @@ The behavior in detail:
+As I have set the `Accept-Language`-Http-Header to `en` for all request and found no other way setting a language for the schema pages I assume it is a lack of internationalization in the C.M.I.
+You could circumvent this by creating map files to map things properly to your language.
+
+If you want to see the possible options of a multi-state field you could open the _schema API page_ with your web browser and click on the object.
+A Popup with an option field will be shown showing all possible options, like in this screenshot:
+
+![screenshot-operation-mode-values](doc/images/operation-mode-values.png)
+
+Please be also aware that there are field having more 'state' values than options, i.E. a manual output override: It has 'Auto/On', 'Auto/Off', 'Manual/On', 'Manual/Off' as state, but only 'Auto', 'Manual/On' and 'Manual/Off' as updateable option.
+You only set it to 'Auto' and the extension On/Off is added depending on the system's current state.
+
+## CoE
+
+### Configure CAN outputs in TAPPS2
+
+You need to configure CAN outputs in your Functional data on the UVR16x2.
+This can be done by using the TAPPS2 application from TA. Follow the user guide on how to do this.
+
+### Configure your CMI for CoE
+
+Now follow the User Guide of the CMI on how to setup CAN over Ethernet (COE).
+Here you will map your outputs that you configured in the previous step.
+This can be accomplished via the GUI on the CMI or via the coe.csv file.
+As the target device you need to put the IP of your openHAB server.
+Don’t forget to reboot the CMI after you uploaded the coe.csv file.
+
 ### TA C.M.I. CoE Connection
 
+The _TA C.M.I. CoE Connection_ has to be manually configured.
+
+This thing reflects a connection to a node behind a specific C.M.I.. This node could be every CAN-Capable device from TA which allows to define a CAN-Input.
+
+| Parameter Label         | Parameter ID    | Description                                                                                                   | Accepted values        |
+|-------------------------|-----------------|---------------------------------------------------------------------------------------------------------------|------------------------|
+| C.M.I. IP Address       | host            | Host name or IP address of the C.M.I                                                                          | host name or ip        |
+| Node                    | node            | The CoE / CAN Node number openHAB should represent                                                            | 1-64                   |
+
+The thing has no channels by default - they have to be added manually matching the configured inputs / outputs for the related CAN Node. Digital and Analog channels are supported. Please read TA's documentation related to the CAN-protocol - multiple analog (4) and digital (16) channels are combined so please be aware of this design limitation.
```

Comment:
```suggestion
The thing has no channels by default - they have to be added manually matching the configured inputs / outputs for the related CAN Node.
Digital and Analog channels are supported.
Please read TA's documentation related to the CAN-protocol - multiple analog (4) and digital (16) channels are combined so please be aware of this design limitation.
```

Each sentence on its own line.

Reply:
Hmm. I didn't touch anything that is not JSON related. I'd rather add a new "Improve documentation"-PR.

Reply:
In that case you should double check this README.md as these lines are changed by this PR.
Not sure about your workflow, but a direct link is here and shows the changes this PR makes to this file:
https://github.com/openhab/openhab-addons/pull/14119/files#diff-bb141e8651142e49d0ecdd584d2652282d0bec24a2bc554045a19a6d40d66028

Reply:
They are changed because I re-organized the readme. I put that into an extra commit, before adding the JSON-Stuff: 3767614b64f3e2811bc0ecf6418160f470dfe289

Reply:
Yes, i see now. Would be nice to have it fixed, either in this PR or in a follow-up. Both is fine by me.

Reply:
Will do in a follow up. :)